merkletree/pad.go

```diff
@@ -5,6 +5,7 @@ import (
 	"errors"
 
 	"github.com/coniks-sys/coniks-go/crypto"
+	conikshasher "github.com/coniks-sys/coniks-go/crypto/hasher/coniks"
```

Comment:
Makes sense! I would still prefer a shorter import. It would saves some typing when using it and it seems more consistent with the generally shorter package names in golang. But it isn't that important.